sw/device/silicon_creator/lib/drivers/lifecycle.c

```diff
@@ -0,0 +1,65 @@
+#include "sw/device/silicon_creator/lib/drivers/lifecycle.h"
```

Comment:
(The tests fail because of missing licence header here.)

Reply:
Fixed.